components/application-mgt/org.wso2.carbon.identity.application.mgt/src/test/resources/repository/conf/identity/identity.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="ISO-8859-1"?>
 <!--
-  ~ Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+  ~ Copyright (c) 2021-2025, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
doesn't need this change

Reply:
addresed with https://github.com/wso2/carbon-identity-framework/commit/8955c37524a997e12c0bb655e9b675508a8a55cc